docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -947,6 +1013,11 @@ Payload objects with the following scopes take the form that follows:
   "data": {
     "type": "sku", // will always be sku
     "id": 461, // ID of the sku
+    "attributes": ["cost_price", "is_free_shipping"],
```

Comment:
I assume the `attributes` and `context` fields only apply to the `store/sku/updated scope`, and not the `store/sku/deleted` and `store/sku/created` scope?

Reply:
K, I changed the doc so that the payload for the `store/sku/updated` scope is different than that of the `store/sku/deleted` and `store/sku/created` scope 👍 

I'm wondering whether we want the doc published before or after CATALOG-9998 is done

Reply:
@jyuricle I believe we also need to remove `store/sku/updated` changes from documentation for now as variant overrides per channel are not yet ready. Correct?

Reply:
Correct